js/storage.js

```diff
@@ -1283,18 +1281,24 @@ Storage.exportTeam = function (team) {
 			text += '' + curSet.nature + ' Nature' + "  \n";
 		}
 
-		if (curSet.ivs) {
+		var ivs = curSet.ivs;
+		if (ivs) {
 			var firstIV = true;
-			for (var j in BattleStatNames) {
-				var val = curSet.ivs[j];
-				if (val === undefined || isNaN(val) || val === 31) continue;
-				if (firstIV) {
-					text += 'IVs: ';
-					firstIV = false;
-				} else {
-					text += ' / ';
+			var all31 = (ivs.hp === 31 && ivs.atk === 31 && ivs.def === 31 && ivs.spa === 31 && ivs.spd === 31 && ivs.spe === 31);
+			if (all31) {
+				text += 'IVs: 31 all';
+			} else {
+				for (var j in BattleStatNames) {
+					var val = curSet.ivs[j];
+					if (val === undefined || isNaN(val) || val === 31) continue;
+					if (firstIV) {
+						text += 'IVs: ';
```

Comment:
This is unnecessary now. `IVs:` will always be displayed, so you can just move this up to where `if (ivs) {` is.

Reply:
Ok